frontend/public/sw.js

```diff
@@ -0,0 +1,73 @@
+self.addEventListener('push', (e) => {
```

Comment:
I'm not familiar with what `self` is. Is this something specific to Nuxt?

Reply:
It's not specific to Nuxt, it's a part of the Service Worker API provided by browsers. Basically it's referencing itself, and adding event listeners to the Service Worker